### DISCUSSION TITLE
Pointing to non-existent redis server

### DISCUSSION THREAD

**Discussion by charleslcso** [ghostfolio/ghostfolio#1556] [Q&A]
Hello,

My server crashed and rebooted, and after that ghostfolio container will stop after a few seconds.

docker logs ID shows that the ghostfolio container is trying to reach redis server at IP 172.19.0.2; but redis server is running at 172.19.0.1!

What can I do to fix this?

**Comment by dtslvr** [comment]
Hi @charleslcso

You can configure the host for _Redis_ via the [environment variable](https://github.com/ghostfolio/ghostfolio#supported-environment-variables) `REDIS_HOST`:

```
REDIS_HOST=172.19.0.1
```

**Reply by charleslcso** [reply]
Hello @dtslvr, I believe the problem is deeper than that.

ghostfolio-development_postgres_1 and ghostfolio-development_redis_1 both stay up and running. ghostfolio-development_ghostfolio_1 will die once web browser tries to access the application.

The container log shows it can't connect to redis server; both postgresql's 5432 and Redis' 6379 are both not accessible via telnet. I believe Docker's network is broken...

Is there a quick and easy way to reset the network back to default setting?
